loopdb/store.go

```diff
@@ -568,6 +568,13 @@ func (s *boltSwapStore) CreateLoopOut(ctx context.Context, hash lntypes.Hash,
 	})
 }
 
+// UpdateLoopOutAssetInfo is unused for the bolt swap store.
+func (db *boltSwapStore) UpdateLoopOutAssetInfo(ctx context.Context, hash lntypes.Hash,
+	asset *LoopOutAssetSwap) error {
+
+	return nil
```

Comment:
Is the bolt implementation expected to handle assets? 

I propose to return an error here ("unimplemented").